### DISCUSSION TITLE
[FEAT] Playground

### DISCUSSION THREAD

**Discussion by VadimasVP** [sunflower-land/sunflower-land#1131] [Ideas]
## Summary of Request
We were recently thinking that we actually lack diversity on the map as far as representation comes, not only models but also skin color/gender and them actually interacting with one another - hence got an idea to make a Playground for children characters to share and play in together.

## Proof of Community Demand

_**Artist chat**_
![unknown](https://user-images.githubusercontent.com/102822587/174568523-25958895-f012-4aaa-947d-139b11ff87a5.png)
_**VIP chat**_
![unknown (1)](https://user-images.githubusercontent.com/102822587/174568526-cbad9b0b-cd3a-4539-ad9d-21ab1443bbd7.png)
_**Farmers chat**_
![unknown (2)](https://user-images.githubusercontent.com/102822587/174568529-5b3ec369-0ae8-409f-ade8-d6b0034e0eb5.png)

## Contributors

Idea @DonatasDee
UI @tiffanydys
Art @VadimasVP

## Technical Requirements

Front-End code
* Add the Playground NFT to the Farmer Goblin in Goblin Land
* Find a place in Farmers Land to which the NFT will be displayed
* Add Playground NFT in Farmers Land
* Make sure to display even with “Visit Mode” ON
* Add metadata for OpenSea requirements

### Backend Updates Needed

* Add Playground NFT with the following requirements:
    * 1000 Wood
    * 0 SFL
    * Supply: 500
    * Withdraw time: 72 Hours
* NFT Release 250 at first wave and another 250 for second wave

### Smart Contract Updates Needed

* Add a function call to mint the Playground NFT

## UI/UX Design Requirements

Playground NFT Interaction 
* When clicked, should display a text modal saying either of the following:
“I just want to play with my friends.”
“I don’t get this war.”
“This swing makes my head spin!”
“I like the goblins!”
“The humans are nice to me!”

(these texts could be randomized on each click)

## Assets Needed

- ✅Final result.

![finalbig](https://user-images.githubusercontent.com/102822587/174475922-57a05436-0522-4853-adfd-d1b2f3223e62.png)

- ✅NPC (animated).

![playing](https://user-images.githubusercontent.com/102822587/174475959-64342308-c581-4c74-94e3-1e5ee9f08ce9.gif)
![swing](https://user-images.githubusercontent.com/102822587/174475962-3287ff9e-ee20-4037-899b-81ced15a1b3b.gif)
![swing1](https://user-images.githubusercontent.com/102822587/174475964-ecb9efe9-5f1a-4391-9221-907bde0a1dc7.gif)

- ✅Decorations.

![decor1](https://user-images.githubusercontent.com/102822587/174475979-367aad45-960f-47c2-af7d-d3efd3b9b23f.png)
![decor](https://user-images.githubusercontent.com/102822587/174475976-5e632bca-a707-4c34-abc6-c2ba7a69f71a.png)
![decor3](https://user-images.githubusercontent.com/102822587/174475980-4e02bc2a-a745-42bb-be1e-bfd3b7a8f491.png)

_**For this to be minted out, we propose to add a tiny buff to it. 
We wouldn't want to come up with the buff ourselves and would like to ask for the core devs what they think is best for this NFT.**_

**Comment by SteveChickenCX** [comment]
I would love to see that ingame, I think it can be fun to burn wood for it!
